common/withdraw_tx.c

```diff
@@ -12,17 +12,29 @@
 
 struct bitcoin_tx *withdraw_tx(const tal_t *ctx,
 			       const struct chainparams *chainparams,
+			       bool allow_rbf,
 			       const struct utxo **utxos,
 			       struct bitcoin_tx_output **outputs,
 			       const struct ext_key *bip32_base,
 			       u32 nlocktime)
 {
 	struct bitcoin_tx *tx;
 	int output_count;
+	/*
+	 * BIP-125: Opt-in Full Replace-by-Fee Signaling
+	 *     https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki
+	 * A transaction is considered to have opted in to
+	 * allowing replacement of itself if any of its
+	 * inputs have an nSequence number less than (0xffffffff - 1).
+	 */
+	/* A sequence of (0xffffffff - 1) signals to use locktime */
+	u32 sequence = BITCOIN_TX_DEFAULT_SEQUENCE - 1;
+	if (allow_rbf)
+		sequence--;
```

Comment:
Isn't this now `0xffffffff - 2` because `BITCOIN_TX_DEFAULT_SEQUENCE` is `0xffffffff` and we subtract twice? That'd mean we always opt in.

Reply:
Right, the intention is to opt in to RBF if 'allow_rbf' is true.

`0xffffffff` is no rbf, no locktime
`0xffffffff - 1` is no rbf, allow locktime (which we set on all withdraws, and now fundchannels)
`0xffffffff - 2` is allow rbf, allow locktime